README.md

```
@@ -73,6 +74,62 @@ Use `@v1` for the newest stable release in major version 1, or `@v1-prerelease`
+    "models": {
+      "review-model": {
+        "input": 1.2,
+        "output": 2,
+        "cache-hit": 0.12,
+        "cache-creation": 0.6
+      }
+    }
+  }
+```
+
+`currency` is preserved as an exact prefix. For example, `$` produces `$0.36`, `USD` produces `USD0.36`, and `USD ` produces `USD 0.36`. Model matching is case-sensitive. The raw SDK model ID is checked first, then its canonical model ID. A model without a matching price remains in the token total, is marked `unpriced`, and makes the estimated cost a lower bound. Incomplete SDK accounting also makes the estimate a lower bound.
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)